src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function dummyMod(m)
```

Comment:
I pushed the uniform scaling.
There's Diagonal and UniformScaling. No DiagonalScaling.
On Apr 18, 2016 17:55, "Lars Ruthotto" <notifications@github.com> wrote:

> In src/InverseSolve/models.jl
> <https://github.com/JuliaInv/jInv.jl/pull/3#discussion_r60159627>:
>
> > @@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
> >  	return f(m),sdiag(df(m))
> >  end
> >
> > +function dummyMod(m)
>
> which version do you use? maybe time for an update!? Here is the
> documentation:
> http://docs.julialang.org/en/release-0.4/manual/linear-algebra/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaInv/jInv.jl/pull/3/files/ace570ed0d40a4b42df3ab8ff7ed285f6dba44f0#r60159627>
>